### DISCUSSION TITLE
Horizontal scroll bars disappearing and NOT reappearing

### DISCUSSION THREAD

**Discussion by Senajs** [mikekazakov/nimble-commander#272] [Q&A]
I have the issue for weeks now that when using a mouse the horizontal scrollbar disappears(as set in MacOS settings) but even when I move the mouse it does NOT re-appear. Spoke with APple support and apart from suggesting to not hide the scrollbar we found that it seems it happens with NON-APple mouse. with the Magic mouse, you can swipe over the mouse and the scrollbar re-appears but with the Logitech I have too it doesn't work at all. Just FYI now - also talking to logitech about this

**Comment by mikekazakov** [comment]
Thanks for the heads-up!
